frontend/src/components/Content.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types'
+import { createUseStyles } from 'react-jss'
+import Theme from './Theme'
+
+const jss = createUseStyles( {
+  content: {
+    margin: '0 auto',
+    width: `calc(100% - 2 * ${Theme.Gutter})`,
+    maxWidth: '1080px',
+    paddingLeft: Theme.Gutter,
+    paddingRight: Theme.Gutter,
+  },
+} )
+
+const Content = ( { children } ) => {
+  const css = jss()
```

Comment:
```suggestion
  const classes= useStyles()
  
```

`classes` here is more idiomatic, as each property is on the object is actually the name of a class - `css`